src/Root/App/styles.css

```
@@ -31,12 +31,15 @@
     --font-weight-bold: 700;
 
     /* COLOR PALETTE */
-    --color-primary: #354052;
-    --color-accent: #008dca;
+    --color-primary: #730fc3;
+    --color-accent: #6558f5;
     --color-danger: #e04656;
     --color-success: #26c681;
     --color-warning: #efe045;
 
+    --tui-color-primary: #730fc3;
+    --tui-color-accent: #6558f5;
```

Please re-use previously defined variable

```suggestion
    --tui-color-primary: var(--color-primary);
    --tui-color-accent: var(--color-accent);
```